docs/api/module.md

```diff
@@ -1,40 +1,42 @@
 # Module
 
-Modules help you structure your state and signals into logical units. You can think of them as namespaces for state and signals.
+Modules are the core building blocks of your application. This is where you define state, signals, other modules etc.
```

Comment:
Maybe better 'submodules' instead of 'other modules'.

Reply:
agree, adjusting :)